### PR TITLE
Add/jitm for agencies in backup cloud

### PR DESCRIPTION
**Pull request by CodeyGuyDylan**
## Proposed Changes

This PR adds a JITM-like banner to the Jetpack Backup Cloud page that shows up only to Agencies/Partners that links to a survey for them to fill out

NOTE: I also made a new directory for all the banners in `my-sites/backup`, so we'll make sure they all still work as well

P2: pbNhbs-5cV-p2
Design convo: p1680716915761929-slack-C0D96691V

## Testing Instructions

1. Checkout this branch
2. Start Calypso Green via `yarn start-jetpack-cloud`
3. If you don't have one already, purchase a Jetpack Backup license and attach it to a site (I used jurassic ninja). Do not enter the server credentials
4. Go to http://jetpack.cloud.localhost:3000/backup/{your-site} 
5. Ensure the server credentials banner still shows up after moving the files to their own folder
Screenshot 2023-04-05 at 1 26 16 PM
6. Make sure the new Agencies banner does not show up yet (unless you're already signed up as a Jetpack Partner)
7. Follow these instructions ( PCYsg-KIj-p2 ) to sign up as a Jetpack Partner. Make sure you choose Partner type of "Agency"
8. Refresh the Jetpack Cloud Backup page and ensure the new banner now shows up
Screenshot 2023-04-05 at 1 28 36 PM
9. Make sure the CTA works and sends them to the correct URL
10. Dismiss the Banner and make sure it doesn't come back on refresh
image

NOTE: If you dismiss the banner and need it to be visible again, in development, you can hover over `JETPACK-CLOUD-DEV`, then Preferences, and at the very bottom there is a checkbox you can un-check to clear the preference
Screenshot 2023-04-05 at 4 10 34 PM

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [x] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/my-sites/backup/banners/agencies-survey-banner.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	return (
+		<Banner
+			className="backup__agencies-survey-banner"
+			callToAction={ translate( 'Take our quick survey' ) }
+			title={ translate( 'Agencies & Builders: Help us make VaultPress Backup better' ) }
+			description={ translate(
+				'What can we do to make your job easier by improving our products & tools?'
+			) }
+			href={ surveyUrl }
+			iconPath={ blogPostComments }
```
Comment (IanRamosC) [inline client/my-sites/backup/banners/agencies-survey-banner.tsx]: I think we could change `blogPostComments` to `blogPostCommentsIcon` or something like that, to make it clear that that variable is an image.

**Review by IanRamosC — APPROVED** [review]
The functionality tests well. First, I tested with a non-agency account, then switched it to agency and it works as expected. Just left a minor comment about the name of an import, but the rest looks good.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7918743

Thank you @CodeyGuyDylan for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7918743).

**Pull request merged**